src/controllers/resource-store/index.tsx

```diff
@@ -226,11 +260,37 @@ export const actions: Actions = {
             data: null,
             error: null,
             expiresAt: getExpiresAt(0),
+            accessedAt: 0,
```

Comment:
If for `maxCache` we remove over limit resources from the cache, should this be changed to remove the resource too instead of set data to `null` and `expiresAt`/`accessedAt` to `0`?

Reply:
Done

Reply:
✅